ext/openssl/openssl.c

```diff
@@ -1064,16 +1064,26 @@ static inline int php_openssl_config_check_syntax(const char * section_label, co
 }
 /* }}} */
 
+static char *conf_get_string_quiet(
+		LHASH_OF(CONF_VALUE) *conf, const char *group, const char *name) {
+	char *str = CONF_get_string(conf, group, name);
+	if (!str) {
```

Comment:
Think the preference in this extension is to do explicit NULL checks which sort of follows the OpenSSL which requires NULL check. I know that PHP wouldn't probably work on platforms where NULL doesn't evaluate to false but it's probably good to have it consistent...